ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/feemarket/LondonFeeMarket.java

```diff
@@ -32,14 +32,13 @@ public class LondonFeeMarket implements BaseFeeMarket {
       GenesisConfigFile.BASEFEE_AT_GENESIS_DEFAULT_VALUE;
   static final long DEFAULT_BASEFEE_MAX_CHANGE_DENOMINATOR = 8L;
   static final long DEFAULT_SLACK_COEFFICIENT = 2L;
-  // required for integer arithmetic to work when baseFee > 0
-  private static final Wei DEFAULT_BASEFEE_FLOOR = Wei.of(7L);
+  private static final Wei DEFAULT_BASEFEE_FLOOR = Wei.of(1L);
```

Comment:
This means we will accept transactions into the pool that are < 7 wei, which I don't think is desirable for the vast majority of cases (where baseFee is permanently > 7).

Reply:
I did this change for consistency with the base fee in the genesis file, but I am fine reverting it since it will apply only in a very corner case

Reply:
reverted